tests/tdk_lambda_genesys.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+from unittest import skipIf
+
+from utils.channel_access import ChannelAccess
+from utils.testing import get_running_lewis_and_ioc
+from utils.ioc_launcher import IOCRegister
+
+# MACROS to use for the IOC
+# Macros may need to be formally set - check dbLoadRecords line in *ioc*_ddevsim_ioc.log file
+MACROS = { "ADDR1": "1", "PORT1": "1"}
+
+
+class OutputMode(object):
+    VOLTAGE = "VOLTAGE"
+    CURRENT = "CURRENT"
+
+
+class Tdk_lambda_genesysTests(unittest.TestCase):
+
+    def setUp(self):
+        self._lewis, self._ioc = get_running_lewis_and_ioc("tdk_lambda_genesys")
+        self.ca = ChannelAccess(default_timeout=10)
+        self.ca.wait_for("GENESYS_01:1:VOLT", timeout=30)
```

Comment:
Why is the timeout so much larger here? If it's for a reason then it might deserve a comment.